src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -187,6 +190,13 @@
       + "specific dialect. All properly-packaged dialects in the JDBC connector plugin "
       + "can be used.";
 
+  public static final String DB_TIMEZONE_CONFIG = "db.timezone";
+  public static final String DB_TIMEZONE_DEFAULT = "UTC";
+  private static final String DB_TIMEZONE_CONFIG_DOC =
+      "Name of the timezone that should be used in the connector when "
+      + "specifying time-based criteria. Defaults to UTC.";
```

Comment:
This is for the sink, so I'm amending my earlier suggestion:

```suggestion
      + "inserting time-based values. Defaults to UTC.";
```

Reply:
:+1